cli/flox/src/commands/environment.rs

```diff
@@ -919,6 +926,17 @@ impl Init {
         );
         Ok(())
     }
+
+    fn validate_env_name(name: &str) -> Result<EnvironmentName, anyhow::Error> {
+        let parsed: Result<EnvironmentName, EnvironmentRefError> = name.parse();
+        if let Err(EnvironmentRefError::InvalidName) = parsed {
+            bail!(format!(
+                "Name '{}' is invalid. Environment names may not contain ' ' or '/'.",
+                name
+            ))
+        }
+        Ok(parsed.unwrap())
+    }
```

Comment:
I think it would be better to improve parsing errors in `cli/flox-rust-sdk/src/models/environment_ref.rs` and possibly map_err if we really need more fancy handling